chapters/synchronization.txt

```diff
@@ -4236,6 +4252,52 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:srcAccessMask member that is not supported by any
+    of the pipeline stages in pname:srcStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:dstAccessMask member that is not supported by any
+    of the pipeline stages in pname:dstStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-release, queue
+    family release operation>>, and its pname:buffer member was created
+    with a sharing mode of ename:VK_SHARING_MODE_EXCLUSIVE, then its
+    pname:dstAccessMask member must: only contain access flags that are
+    supported by any of the pipeline stages in pname:dstStageMask, as
+    specified in the <<synchronization-access-types-supported, table of
+    supported access types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-acquire, queue
+    family acquire operation>>, and its pname:buffer member was created
+    with a sharing mode of ename:VK_SHARING_MODE_EXCLUSIVE, then its
+    pname:srcAccessMask member must: only contain access flags that are
```

Comment:
This might be insufficient to replace the below removed VUs.
It should  say "valid access flags" otherwise non-enabled extension access masks would still be valid here per the table.

Reply:
Sticking the word "valid" on it doesn't really change anything here, this suffers from the existing "don't validate extensions explicitly spec" issue. The correct fix would be to add an explicit VU banning each relevant access type explicitly when its extension/feature is not enabled. And fixing that is definitely beyond the scope of this PR...

Reply:
Hm, I somehow assumed that's what "valid flag" means. I guess not.